README.md

```diff
@@ -0,0 +1,16 @@
+# Community
+
+Here you can find community content around Cilium.
+
+## Contribute
+
+Check out the [contributor ladder](CONTRIBUTOR-LADDER.md) for different ways to get involved and level up within the project. You can see different roles within the project in the [contributor roles](CONTRIBUTOR-ROLES.md). If you have already contributed to the project in any way, please add yourself to the [contributor file](CONTRIBUTORS.md).
+
+# Socializing
+
+Connect with the community through:
+
+* Follow us on [Twitter](https://twitter.com/ciliumproject) and our [blog](https://cilium.io/blog/)
```

Comment:
Do we have a Mastodon alternative, as well?

Reply:
No, but we do have LinkedIn so I'll add that